src/DotNetWorker.Grpc/protobuf/src/proto/FunctionRpc.proto

```diff
@@ -431,6 +431,7 @@ message TypedData {
     CollectionDouble collection_double = 10;
     CollectionSInt64 collection_sint64 = 11;
     ModelBindingData model_binding_data = 12;
+    CollectionModelBindingData collection_model_binding_data = 13;
```

Comment:
Changes from this file will be removed before merging once [this](https://github.com/Azure/azure-functions-dotnet-worker/pull/1206) PR is completed.